.github/workflows/tm-load-test.yml

```diff
@@ -0,0 +1,64 @@
+name: Setup chain and apply tm load test
+on:
+  pull_request:
+    branches: [dev]
+
+
+jobs:
+  amd64:
+    name: Setup Chain.
+    runs-on: ubuntu-latest
+    if: github.ref != 'refs/heads/config'
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+        
+      - name: Generate build number
+        id: buildnumber
+        uses: einaregilsson/build-number@v3
+        with:
+          token: ${{secrets.github_token}}
+
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v1
+        with:
+          version: latest
+
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      - name: Build and push image on dockerhub
+        run: docker buildx build --tag  puneet166/cosmos-node:v-${{ steps.buildnumber.outputs.build_number }} --file  deploy/tm-load-test/Dockerfile --platform  linux/amd64 --push --progress  tty .
```

Comment:
puneet166/cosmos-node what does this name mean? And why do we need it to be pushed?